### DISCUSSION TITLE
AKHQ with self-signed certificates?

### DISCUSSION THREAD

**Discussion by ay-b** [tchiotludo/akhq#864] [Q&A] [answered]
Hey. 
I need to deploy akhq to the kafka-dev env which isn't backed up with valid certificates and akhq denies access there claiming about invalid certificates. 
How can I make it work?
Probably it's about `ssl.[something]` option in properties, but what?

Thanks.

**Comment by tchiotludo** [comment]
seems that this tutorial as your guide : https://piotrminkowski.com/2019/04/25/micronaut-tutorial-security/
talking about self signed by micronaut or custom self signed 
or you can follow the micronaut doc [here](https://docs.micronaut.io/latest/guide/index.html#https)

**Reply by ay-b** [reply]
Nope, that tutorial is how to protect akhq with SSL (or I'm missing something?). But I need akhq to connect to Kafka cluster that is using self-signed certs. Is it possible?

**Comment by tchiotludo** [comment, marked as answer]
In that case, you need to follow this tutorial : https://docs.confluent.io/2.0.0/kafka/ssl.html#configuring-kafka-clients

Akhq is a simple java app with basic kafka consumer / producer

**Reply by ay-b** [reply]
Found out that getting a true cert is way simpler. 
Anyway, thanks a lot for the tool!
